docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -285,37 +285,44 @@ The master supports the following configuration settings:
 
       -  ``job_storage_root``: The shared directory where job-related files will be stored. It is
          where the needed Determined executables are copied to when the experiment is run, as well
-         as where the Slurm ``sbatch`` script and log files are created. This directory must be
-         visible to the launcher and from the compute nodes.
+         as where the Slurm/PBS scripts and log files are created. This directory must be writable
+         by the launcher and accessible to the compute nodes to enable the logs to be written from
+         the submitted batch jobs.
 
-      -  ``path``: The ``PATH`` for the launcher service so that it is able to find the Slurm,
+      -  ``path``: The ``PATH`` for the launcher service so that it is able to find the Slurm, PBS,
          Singularity, Nvidia binaries, etc., in case they are not in a standard location on the
          compute node. For example, ``PATH=/opt/singularity/3.8.5/bin:${PATH}``.
 
       -  ``ld_library_path``: The ``LD_LIBRARY_PATH`` for the launcher service so that it is able to
-         find the Slurm, Singularity, Nvidia libraries, etc., in case they are not in a standard
-         location on the compute node. For example,
+         find the Slurm, PBS, Singularity, Nvidia libraries, etc., in case they are not in a
+         standard location on the compute node. For example,
          ``LD_LIBRARY_PATH=/cm/shared/apps/slurm/21.08.6/lib:/cm/shared/apps/slurm/21.08.6/lib/slurm:${LD_LIBRARY_PATH}``.
 
       -  ``tres_supported``: Indicates if ``SelectType=select/cons_tres`` is set in the Slurm
          configuration. Affects how Determined requests GPUs from Slurm. The default is true.
 
-      -  ``gres_supported``: Indicates if ``GresTypes=gpu`` is set in the Slurm configuration, and
-         nodes with GPUs have properly configured GRES indicating the presence of any GPUs. The
-         default is true. When false, Determined will request slots_per_trial nodes and utilize only
-         GPU 0 on each node. It is the user's responsibility to ensure that GPUs will be available
-         on nodes selected for the job using other configurations such as targeting a specific
-         resource pool with only GPU nodes or specifying a Slurm constraint in the experiment
-         configuration.
+      -  ``gres_supported``: Indicates if GPU resources are properly configured in the HPC workload
+         manager.
 
-      -  ``partition_overrides``: A map of Slurm partition names to partition-level overrides. For
+         For PBS, the ``ngpus`` option can be used to identify the number of GPUs available on a
+         node.
+
+         For Slurm, ``GresTypes=gpu`` is set in the Slurm configuration, and nodes with GPUs have
```

Comment:
The "and nodes with GPUs have properly configured GRES" part is confusing. The nodes themselves don't configure anything, nor is anything configured on the nodes.  The nodes do need the proper NVIDIA or ROCM binaries, libraries, and drivers, though.

Reply:
Not node configuration directly, the gres for the node needs to be configured to reflect the GPUs on the nodes.